tools/clang_tidy/lib/clang_tidy.dart

```diff
@@ -228,6 +228,24 @@ class ClangTidy {
     return _ComputeJobsResult(jobs, sawMalformed);
   }
 
+  Iterable<String> _trimGenerator(String output) sync* {
```

Comment:
You can test this by making it public, adding the `@visibleForTesting` annotation, and writing a unit test in `clang_tidy/test/clang_tidy_test.dart`

Reply:
done